sphinx/util/docutils.py

```diff
@@ -182,10 +183,46 @@ def using_user_docutils_conf(confdir: Optional[str]) -> Generator[None, None, No
             os.environ['DOCUTILSCONFIG'] = docutilsconfig
 
 
+@contextmanager
+def du19_footnotes() -> Generator[None, None, None]:
```

Comment:
Is there any reason to patch docutils instead of overriding methods in `sphinx/writers/html5.py`? I think overriding is a bit simpler than patching. Either is okay to me.

Reply:
Makes it easier to remove the patch when we drop support for DU 18 was my thought.

A

Reply:
Okay, let's go with this.